pkg/virt-api/webhooks/validating-webhook/validating-webhook.go

```diff
@@ -125,6 +125,16 @@ func validateDisks(field *k8sfield.Path, disks []v1.Disk) []metav1.StatusCause {
 				Field:   field.Index(idx).Child("name").String(),
 			})
 		}
+
+		// Reject Floppy disks
+		if disk.Floppy != nil {
```

Comment:
The discussion in https://bugzilla.redhat.com/show_bug.cgi?id=1674443 is somewhat ambiguous to me. Was the final decision to outright reject floppies altogether (as is done here), or to only reject them for q35+ machine types?

Since KubeVirt only supports q35 and future, it's not really clear to me that there's a distinction here, but just wanted to double check.

Reply:
As clarified offline, agreement is to always reject floppy disks